docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,82 @@
+# Relative layout
+
+The `RelativeLayout` is a layout class, in which each side of children can be specified as relative to sibling view or parent.
+
+The `Target` and the `RelativeOffset` properties are used to define a relationship with each other. The following figure shows the relative layout position rectangles with `Target` and `RelativeOffset`. The arrows in the preview indicate where the `LeftTarget` and the `RightTarget` is pointing.
+
+When the `Target` and the `RelativeOffset` properties are set as default, the child `RedRect` is aligned to the left side of the parent layout. The `BlueRect` is aligned to the right side of the parent layout by the `HorizontalAlignment`. The left side of `GreenRect` is set to be aligned with `RedRect` and the right side is set to be aligned with `BlueRect`. It fills the space between `RedRect` and `BlueRect` by using `FillHorizontal`. The `YellowRect` represents that each side can be set sibling view and parent layout:
+
+**Figure 1: Preview of relative layout example with `Target` and `RelativeOffset`**
+
+![relativeLayoutExamplePreview](./media/relativeLayoutExamplePreview.png)
```

Comment:
Can we move the figure after line 5, that seems to be the more appropriate place, followed by the description.

Reply:
Figure 2 is also moved.